UI/index.html

```diff
@@ -0,0 +1,386 @@
+<!DOCTYPE html>
+<html lang = "en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="/Users/vladimirbusygin/Desktop/UP/UI/styles.css">
+    <title>YM Market</title>
+</head>
+
+<body>
+    <header class="header">
+        <div class="header_name_first">YM</div> 
+        <div class="header_name_second">Market</div> 
+
+        <button class="sign_in_button">Sign in</button> 
+        <button class="exit_button">Exit</button> 
+    </header>     
+    <div class="scroll_area_main">
+        <div class="post_popular_header">Popular posts</div>
+        
+        <div class="scroll_area_popular">
+            <div class="post">
+                <div class="post_name">Chihuahua puppies</div>
+                <div class="post_vendor_name">Sashka sobachnik</div>
+                <a class="post_link" href="busuni.com">CLICK HERE</a>
+                <div class="post_hashtags">#chihuahua#minsk</div>
+                <div class="post_information">Cool puppies to good hands</div>
+                <div class="post_price">100 millions</div>
+                <div class="post_period">10%</div>
+                <div class="post_date">09.03.21 - 15.03.21</div>
+                <div class="post_photo_area">
+                    <div class="post_add_photo">
+                        <img src="https://кутята.рф/media/breed/Dogs_White_Chihuahua.jpg">
+                    </div>
+                    <div class="post_add_photo">
+                        <img src="https://prosobak.com/wp-content/uploads/2015/07/chihuahua-samaea-malenikaya-sobaca-v-mire-1024x1024-500x500.jpg">
+                    </div>
+                    <div class="post_add_photo">
+                        <img src="https://www.royal-canin.ru/upload/iblock/c41/ekst.jpg">
+                    </div>
+                    <div class="post_add_photo">
+                        <img src="https://domashnij-zapovednik.com/wp-content/uploads/2018/11/mal.wnt5Y.jpg">
+                    </div>
+                </div>
+                <button class="button_add_rate">Add rate</button>
+                
+                <button class="button_show_comments">Comments</button>
```

Comment:
According to the task, latest comments should be shown for each post

Reply:
By "Comments" button, I meant in the future, when I click on it, a list of latest comments will drop down.